Source/Scene/Cesium3DTile.js

```diff
@@ -295,66 +323,62 @@ define([
         });
     }
 
-    function createDebugBox(box, color) {
-        var geometry = new RectangleOutlineGeometry({
-            rectangle : new Rectangle(box[0], box[1], box[2], box[3]),
-            height : box[4],
-            extrudedHeight: box[5]
-         });
-        return createDebugPrimitive(geometry, color);
-    }
-
-   function createDebugOrientedBox(obb, color) {
-        var geometry = BoxOutlineGeometry.fromDimensions({
-            dimensions: new Cartesian3(2.0, 2.0, 2.0)
-        });
-        return createDebugPrimitive(geometry, color, Matrix4.fromRotationTranslation(obb.halfAxes, obb.center));
+    function createDebugVolume(boundingVolume, color) {
+        var geometry;
+        var modelMatrix = new Matrix4(
+            1, 0, 0, 0,
+            0, 1, 0, 0,
+            0, 0, 1, 0,
+            0, 0, 0, 1
+        );
```

Comment:
Thanks -- I stumbled on the immutability of Matrix4.IDENTITY, didn't figure out I should clone it :)